posts/2021-07-08-sig-bugs.adoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "A number of significant bug fixes in Open Liberty 21.0.0.7"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: A number of significant bug fixes in Open Liberty 21.0.0.7 - OpenLiberty.io
+seo-description: Open Liberty 21.0.0.7 comes with out any new features but a number of signifcant bug fixes.
+blog_description: Open Liberty 21.0.0.7 comes with out any new features but a number of signifcant bug fixes.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= A number of significant bug fixes in Open Liberty 21.0.0.7
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+// tag::intro[]
+
+Open Liberty 21.0.0.7 comes with out any new features but a number of signifcant bug fixes. 
```

Comment:
It appears that the majority of the work this release went into beta features.  Perhaps we should call that our here with a link to the beta blog.  It's also a sneak peak for whats coming next month.

Reply:
Changed.